### ISSUE TITLE
Feature request: Select all (joint points)

### ISSUE THREAD

**Issue by Woisek** [fkunn1326/openpose-editor#36]
I often re-create poses from full body shots. For this, I mostly scale the image up and refine the pose. Then, mostly I want to reduce to, say, a portrait size. I shrink the preview image to the final size (536x768). But after that, I can't select all joints to resize it to match.
So, it would be nice to have a button "Select all" to get all joints selected, no matter if outside the frame or not.
Thanks.

**Comment by testFaze** [comment]
if you are 'refining the pose' in this extension, you can simply click hold/drag select the whole figure (or part of it) and then you will get corner handles etc. that you can drag to resize - like doing powerpoint or something.  the rotate doesn't work yet

**Comment by Woisek** [comment]
@testFaze 
That doesn't work for joints outside the frame, thus the problem I described.

**Comment by testFaze** [comment]
Ok got you.  I guess it's (literally) a barebones extension at the moment.  There are lots of fairly obvious things it could or should be able to do.  Perhaps the author will build on what's there or something else will come along.  As well as the out of bounds selection issue and rotation, in order of difficulty they'd seem to me: ability to read in a pose png (the coloured bones on a black field, not a picture); hands poses; 2d-IK logic; and finally a basic but functional 3d-manipulation to-2d-pose interface (that may be really hard, not sure?) so you don't need blender or Daz.  At rate of SD/AI progress all could happen quite fast, maybe we just ask ChatGPT to write the script.  The rate of enhancements is spectacular but great to see it still lags our expectations (I'm not criticising anyone, this is all great).

**Comment by Woisek** [comment]
Yes, I would agree with you. And what I also experienced is (but that is not necessarily a problem of OpenPose Editor) the lack of definition, which part of the body is "in front/in back". Some poses result in an abnormal twisting of the body because the model is not clear what is in front and what is behind becasue of the missing Z axis (depth). I think controlnet should really enhance openpose to get proper poses done.

**Comment by robonxt** [comment]
Would love to see this fixed as well. Loading from preset seems to make the model load outside of the canvas, and I am unable to drag the parts back successfully without having a few joints (usually the eyes and one wrist) way outside of the canvas
